source/reference/versioning.txt

```diff
@@ -63,11 +56,9 @@ Considerations
 Same Server Version
 ~~~~~~~~~~~~~~~~~~~
 
-{+c2c-product-name+} only supports syncing between the same version of
-MongoDB Server. The major and minor :ref:`server version numbers
-<release-version-numbers>` must be the same on both clusters.
-Starting in ``mongosync`` 1.6.0, the source and destination cluster
-can use different patch releases of the same major and minor version.
+.. versionchanged:: 1.7
+
+.. include:: /includes/cross-version-sync.rst
```

Comment:
Should we include this twice on the same page? I might be include to remove it above and leave it here for clarity.

Reply:
Ah that makes sense. Looks like I overlooked it when I was going through the file for any redundant includes!